quadratic-core/src/color.rs

```
@@ -24,6 +24,15 @@ impl Default for Rgba {
 }
 
 impl Rgba {
+    pub fn new(red: u8, green: u8, blue: u8, alpha: u8) -> Self {
+        Self {
+            red,
+            green,
+            blue,
+            alpha,
+        }
+    }
+
     pub fn color_from_str(color_str: &str) -> Result<Self, ParseIntError> {
         // TODO(jrice): serde
         assert_eq!(&color_str[0..=0], "#");
```

I know this is legacy, but not sure why jabob added an assertion instead of returning an error to avoid the panic.